test/distributed/test_c10d.py

```diff
@@ -273,6 +273,30 @@ def test_address_already_in_use(self):
             store1 = c10d.TCPStore(addr, port, 1, True)  # noqa: F841
             store2 = c10d.TCPStore(addr, port, 1, True)  # noqa: F841
 
+    def _test_numkeys_delkeys(self, fs):
+        # We start off with one init key in the store to coordinate workers
+        self.assertEqual(fs.num_keys(), 1)
+        fs.add("key", 1)
+        fs.add("key", 2)
+        fs.add("key", 3)
+        fs.set("key0", "value0")
```

Comment:
curious, what's the difference between add and set?

Reply:
`add` is used to increment a counter associated with the key, whereas `set` can be used to insert a key-value pair into the store.